portfolio/src/main/webapp/script.js

```diff
@@ -86,7 +86,23 @@ function changePattern(pattern) {
     }
   }
 }
+function createCommentElement(eachComment) {
+    const divElement = document.createElement('div');
```

Comment:
Should be consistent with the usage of single or double quotes, because you have used double quotes below.

Reply:
I have just modified the strings to use double quotes.